Framework/Nexus/inc/MantidNexus/MuonNexusReader.h

```diff
@@ -85,9 +86,10 @@ class DLLExport MuonNexusReader {
   int t_ntc1; ///< number of time channels in time regime 1
   int t_nper; ///< number of periods in file (=1 at present)
   // for nexus histogram data
-  float *corrected_times;                ///< temp store for corrected times
-  int *counts;                           ///< temp store of histogram data
-  int *detectorGroupings;                ///< detector grouping info
-  int numDetectors;                      ///< detector count
-  std::string getInstrumentName() const; ///< return instrument name
+  float *corrected_times;                         ///< temp store for corrected times
```

Comment:
As we are in this code area can I ask we make some effort towards modernization and remove using plain C arrays and use vectors by default?